docs/src/proposals/scrambled-tx.md

```diff
@@ -0,0 +1,335 @@
+---
+title: Scrambled Transaction to combat against MEV
+---
+
+### Program Statement
+
+The existence of MEV isn't ideal and costly.
+
+MEV motivates transaction reordering, censoring and copying. Also, it causes
+cluster destabilization because of incentivised forking.
+
+The root problem is that there is no rule in the pursuit of MEV frontier, except
+economic rationality. While this anarchism is welcomed, stemmed from the
+blockchain philosophy, it's also desirable for such an activity to economically
+impractical in the first place.
+
+Any temporal market imbalance should be corrected strictly on the grounds of a
+fair game, basically the golden first-come-first-served rule (with optional
+bidding for priority).
+
+Note that this proposal doesn't fully mitigate one of MEV types, unlike all
+other fully-neutralized types: back-running. Back-running can be exercised
+while the golden rule is strictly and fairly enforced.
+
+### Summary of Proposed Solution
+
+This proposal tries to combat against MEV, at the cost of (slight) global
+induced latency of cluster as a whole.
```

Comment:
Perhaps also important to point out, this proposal is not only addressing MEV but is also a potential solution for validator-based transaction censorship (including transactions related to consensus, such as validator nomination and perhaps leader scheduling), isn't that right ?

Reply:
yes, exactly. thanks for spelling out.